shared/common-adapters/text.desktop.js

```diff
@@ -133,9 +147,9 @@ export const styles = {
   },
   textBodySmall: {
     ...textCommon,
+    color: globalColorsDZ2.black40,
     fontSize: 14,
     lineHeight: '19px',
-    opacity: 0.4,
```

Comment:
fine with me, I did it because I didn't want people setting an opacity _and_ having to reset the color to black,

but since we are just using this set of opacities it's probably fine to not touch the opacity field and just use the color field.